src/egon/data/datasets/power_etrago/match_ocgt.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Module containing the definition of the open cycle gas turbine links
+"""
+from geoalchemy2.types import Geometry
+from scipy.spatial import cKDTree
+import numpy as np
+import pandas as pd
+
+from egon.data import db
+from egon.data.datasets.etrago_setup import link_geom_from_buses
+
+
+def insert_open_cycle_gas_turbines(scn_name="eGon2035"):
+    """Insert gas turbine links in egon_etrago_link table.
+
+    Parameters
+    ----------
+    scn_name : str
+        Name of the scenario.
+    """
+
+    # Connect to local database
+    engine = db.engine()
+
+    # create bus connections
+    gdf = map_buses(scn_name)
+
+    # create topology column (linestring)
+    gdf = link_geom_from_buses(gdf, scn_name)
+    gdf["p_nom_extendable"] = False
+    carrier = "OCGT"
+    gdf["carrier"] = carrier
+
+    # Delete old entries
+    db.execute_sql(
+        f"""
+        DELETE FROM grid.egon_etrago_link WHERE "carrier" = '{carrier}'
```

Comment:
Will there be 'OCGT' components at the foreign buses? If yes, be careful to not delete them here.

Reply:
Good, point, this is related to #567. I will try to add the methods accordlingly until tomorrows CI run.